src/analysis/normed_space/normed_group_quotient.lean

```diff
@@ -492,4 +492,36 @@ begin
   { exact ⟨0, f.ker.zero_mem, by simp⟩ }
 end
 
+lemma lift_bound_by {N : Type*} [semi_normed_group N] (S : add_subgroup M)
+  (f : normed_group_hom M N) (hf : ∀ s ∈ S, f s = 0)
+  {c : ℝ≥0} (fb : f.bound_by c) :
+  (lift S f hf).bound_by c :=
```

Comment:
I realize that I was just being completely ignorant with this `bound_by` definition. Would you mind phrasing this in terms of the operator norm?